### PR TITLE
Qobodevelop

### PR DESCRIPTION
**Pull request by constantinoschristoforou**

### CODE REVIEW

File: .idea/cake_config_setting_v0_5.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mamchenkov) [inline .idea/cake_config_setting_v0_5.xml]: @constantinoschristoforou please configure your IDE to NOT create files like this.  Also please remove this one too.

File: shared/shortcodes/ajax-functions.php [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+
+
+echo do_shortcode( $shortcode ) . $extra_content; ?>
+
+<?php // echo "<br>". $shortcode; ?>
+
+
+
+
+<?php    die();
```
Comment (mamchenkov) [inline shared/shortcodes/ajax-functions.php]: @constantinoschristoforou why do you have code after die() ?

File: shared/shortcodes/inbound-shortcodes.php [hunk 3]
```diff
@@ -750,14 +750,19 @@ static function inbound_forms_header_area()
 						</div>
 
 						<div id="inbound-shortcodes-preview-wrap">
-							<div id="inbound-shortcodes-preview-head">
+							<div  id="inbound-shortcodes-preview-head">
```
Comment (mamchenkov) [inline shared/shortcodes/inbound-shortcodes.php]: @constantinoschristoforou why is this necessary?

File: shared/shortcodes/js/shortcodes.js [hunk 4]
```diff
@@ -295,13 +301,14 @@
 			                   });
 						jQuery('body').trigger("inbound_forms_data_ready"); // Trigger custom hook
 
+
 			            }
 			    },
 				update_fields : function () {
-								var insert_form = jQuery("#inbound_shortcode_insert_default").val();
+                    var insert_form = jQuery("#inbound_shortcode_insert_default").val();
```
Comment (mamchenkov) [inline shared/shortcodes/js/shortcodes.js]: @constantinoschristoforou please format nicely

File: shared/shortcodes/ajax-functions.php (hunk 2 above)
Comment (constantinoschristoforou) [inline shared/shortcodes/ajax-functions.php]: The die function is called on the last line of the function

Comment (mamchenkov) [inline shared/shortcodes/ajax-functions.php]: @constantinoschristoforou maybe some coding style / formatting could make it more obvious, but OK for now :)
